include/odp/api/spec/timer.h

```diff
@@ -287,80 +361,94 @@ odp_timer_t odp_timer_alloc(odp_timer_pool_t tpid,
  * The timeout event for an expired timer will not be returned. It is the
  * responsibility of the application to handle this timeout when it is received.
  *
- * @param tim      Timer handle
+ * @param timer      Timer
+ *
  * @return Event handle of timeout event
  * @retval ODP_EVENT_INVALID on failure
  */
-odp_event_t odp_timer_free(odp_timer_t tim);
+odp_event_t odp_timer_free(odp_timer_t timer);
 
 /**
- * Set a timer (absolute time) with a user-provided timeout event
+ * Set (or reset) a timer with absolute expiration time
  *
- * Set (arm) the timer to expire at specific time. The timeout
- * event will be enqueued when the timer expires.
+ * This function sets a timer to expire at a specific time. If the timer is
+ * already running (set and not yet expired), the function updates (resets) it
+ * with a new expiration time and optionally with a new event. A successful
+ * reset operation with a new event returns the old event.
  *
- * @param tim      Timer
- * @param abs_tck  Expiration time in absolute timer ticks
- * @param[in,out] tmo_ev  Reference to an event variable that points to
- * timeout event or NULL to reuse the existing timeout event. Any existing
- * timeout event that is replaced by a successful set operation will be
- * returned here.
+ * The user provided event can be of any event type, but only ODP_EVENT_TIMEOUT
+ * type events (odp_timeout_t) carry timeout specific metadata. Also timer
```

Comment:
`Also timer...`, missing comma or replace with for example` Furthermore,`.

Reply:
Fixed